subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -448,7 +449,14 @@ u8_t ll_adv_enable(u8_t enable)
 	lll = &adv->lll;
 
 	pdu_adv = lll_adv_data_peek(lll);
-	if (pdu_adv->tx_addr) {
+
+#if defined(CONFIG_BT_CTLR_PRIVACY)
```

Comment:
Isn't it always own addr type?

Reply:
yes, tx_addr is 1 bit own addr type, public or random.